build.gradle

```diff
@@ -47,6 +48,7 @@ dependencies {
     testImplementation "nl.jqno.equalsverifier:equalsverifier:3.7.2"
 }
 
+
```

Comment:
unneeded change.

Reply:
strange i already removed this whiteline and did amend it to the commit, why is it still there?